Backpacks.cs

```diff
@@ -165,7 +167,7 @@ private object CanAcceptItem(ItemContainer container, Item item)
 
             Backpack backpack = _backpacks.Values.FirstOrDefault(b => b.IsUnderlyingContainer(container));
 
-            if (backpack != null)
+            if (backpack != null && !permission.UserHasPermission(backpack.OwnerId.ToString(), NoBlacklistPermission))
```

Comment:
Possibly cache the owner ID as a string on the backpack, just to avoid creating new strings every time an item is moved.

Reply:
Updated.